src/resultsutil.h

```diff
@@ -51,13 +54,23 @@ void setupHeaderView(QTreeView* view);
 
 void connectFilter(QLineEdit* filter, QSortFilterProxyModel* proxy);
 
-void setupTreeView(QTreeView* view, QLineEdit* filter, QAbstractItemModel* model, int initialSortColumn, int sortRole,
```

Comment:
this should be kept as an implementation detail, and should take the proxy model as arg (but not the templated type, just a `QAbstractProxyModel*`)